react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { timesClicked: 0 }
+    this.handleClick = this.handleClick.bind(this)
+  }
+  handleClick() {
+    this.setState({ timesClicked: this.state.timesClicked+=1 })
```

Comment:
We never do anything that would assign a value to a state property. Instead, think of it and calculating a new value and _replacing_ the old value.

`+=` is an assignment. use `this.state.timesClicked + 1` instead.